.github/workflows/python_tests.yml

```diff
@@ -38,28 +38,15 @@ jobs:
       - uses: actions/cache@v2
         with:
           path: ~/infogami_python_venv
-          key: ${{ runner.os }}-venv1-${{ hashFiles('*requirements.txt') }}
+          key: ${{ runner.os }}-venv-${{ hashFiles('requirements.txt') }}-${{ hashFiles('requirements_test.txt') }}
```

Comment:
```suggestion
          key: ${{ runner.os }}-venv-${{ hashFiles('requirements*.txt') }}
```
Would that work?

Reply:
I don't think so that's how globs work, but we could try. I am happy to be wrong about that :)